web/src/test/java/org/springframework/security/web/session/SessionManagementFilterTests.java

```diff
@@ -46,9 +49,11 @@
 /**
  * @author Luke Taylor
  * @author Rob Winch
+ * @author Mark Chesney
  */
 public class SessionManagementFilterTests {
 
+	@BeforeEach
```

Comment:
`SecurityContextHolderAwareRequestWrapperTests` and a number of other tests don't clear the context after each/all tests, so it becomes necessary to counteract side effects that crop up when such tests are run together.

```java
…
package org.springframework.security.web.servletapi;
…
public class SecurityContextHolderAwareRequestWrapperTests {

	@BeforeEach
	public void tearDown() {
		SecurityContextHolder.clearContext();
	}
…
}
```